lib/fastlane/plugin/wpmreleasetoolkit/actions/common/get_prs_list_action.rb

```diff
@@ -60,20 +45,16 @@ def self.available_options
                                        description: 'The remote path of the GH repository on which we work',
                                        optional: false,
                                        type: String),
-          FastlaneCore::ConfigItem.new(key: :start_tag,
-                                       description: 'The tag from which the report starts',
-                                       optional: false,
-                                       is_string: true),
-          FastlaneCore::ConfigItem.new(key: :end_tag,
-                                       description: 'The tag to which the report ends',
-                                       optional: true,
-                                       default_value: '.',
-                                       is_string: true),
           FastlaneCore::ConfigItem.new(key: :report_path,
                                        env_name: 'GHHELPER_REPORTPATH',
                                        description: 'The path of the report file',
                                        optional: false,
                                        is_string: true),
+          FastlaneCore::ConfigItem.new(key: :tag,
```

Comment:
Given that what we indeed use for the search is the milestone not the (potentially not-yet-existing) tag, I think it would be better to name this param either `milestone` or `version` rather than `tag`.

And maybe also mention in the ConfigItem's description that this will use the milestone — because sure for our case in WC/WP/SN we name our milestones by the name of the version, but that might not be the case for other users of our toolkit (eg non-a8c, but also even A12s like if Tumblr/Orangina starts to use our toolkit but doesn't follow the exact same conventions at first, etc)

Reply:
Looking at the codebase more closely, I see we have a `get_milestone(release)` helper method, so alternatively we could make this param be named `version` and then use that helper method to find the milestone corresponding to that version before calling your other helper method to list PRs.

…though I'd argue that the implementation for that `get_milestone(version)` method is similarly tied to our conventions (and also its implementation could be improved via a `.reverse.find` instead of the `.each` to make it more functional… but I digress…)

Reply:
Yeah, I was initially going to re-work `get_milestone`, but it needed more refactoring than I wanted to spend on it to fix _this_ issue. I feel like it could definitely be replaced by something a bit simpler for sure :)

Reply:
Yeah let's keep the expected param for this be the milestone directly (and not rely on `get_milestone` for now), I agree.